Fix the default Tasklist configuration of application.yml file

Default configuration:
```
camunda.tasklist:
  username: anUser
  password: aPassword
  roles:
    - OWNER
    - USER
```

The following error occurred using the default configuration:
```
Caused by: java.lang.IllegalArgumentException: USER does not exists as Role in [OWNER, OPERATOR, READER]
        at io.camunda.tasklist.webapp.security.Role.fromString(Role.java:23) ~[classes!/:?]
        at java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:195) ~[?:?]
        at java.util.ArrayList$ArrayListSpliterator.forEachRemaining(ArrayList.java:1654) ~[?:?]
        at java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:484) ~[?:?]
        at java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:474) ~[?:?]
        at java.util.stream.ReduceOps$ReduceOp.evaluateSequential(ReduceOps.java:913) ~[?:?]
        at java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234) ~[?:?]
        at java.util.stream.ReferencePipeline.collect(ReferencePipeline.java:578) ~[?:?]
```

Fix the default Tasklist configuration:
```
camunda.tasklist:
  username: anUser
  password: aPassword
  roles:
    - OWNER
    - OPERATOR
```

--- a/docs/self-managed/tasklist-deployment/authentication.md
+++ b/docs/self-managed/tasklist-deployment/authentication.md
@@ -23,7 +23,7 @@ camunda.tasklist:
   password: aPassword
   roles:
     - OWNER
-    - USER
+    - OPERATOR
 ```
 
 On Tasklist startup, the user is created if they did not exist before.

--- a/docs/self-managed/tasklist-deployment/configuration.md
+++ b/docs/self-managed/tasklist-deployment/configuration.md
@@ -280,6 +280,10 @@ camunda.tasklist:
   # Default: demo/demo
   #username:
   #password:
+  #roles:
+  #  - OWNER
+  #  - OPERATOR
+  
   # ELS instance to store Tasklist data
   elasticsearch:
     # Cluster name

